### DISCUSSION TITLE
crystalizer location

### DISCUSSION THREAD

**Discussion by ghost** [wwmm/easyeffects#1473] [Q&A]
I can't found the crystalizer plugin, can someone help me please?
I want to use the crystalizer to produce some songs, but i cant found it. Im using linux btw

**Comment by wwmm** [comment]
The crystalizer that is available in EasyEffects is a built-in plugin that at this moment can be used only on it. It is not like the usual LV2 plugins that can be loaded on Audio hosts.

**Reply by wwmm** [reply]
>can be used only on it

By this I mean technical reasons and not things like license. I did not write a standalone plugin.

**Reply by ghost** [reply]
yeah dont worry, mmm u dont know if theres some alternative to the effect?

**Reply by wwmm** [reply]
In the past I used the code of a ffmpeg plugin that is called crystalizer but it has a different algorithm. And I am not sure if there is a way to use it in Plugin hosts.

**Reply by ghost** [reply]
thx <3

**Reply by ghost** [reply]
i think, im gonna search made the effect with other effects, if u found how to do it a plugin message me, thx for u attention <3
